space/space_test.go

```diff
@@ -104,9 +104,21 @@ func (test *repoBBTest) TestDelete() {
 
 func (test *repoBBTest) TestList() {
 	_, orgCount, _ := test.list(nil, nil)
-	expectSpace(test.create(testSpace), test.requireOk)
-	_, newCount, _ := test.list(nil, nil)
-	assert.Equal(test.T(), orgCount+1, newCount)
+	newSpace, err := expectSpace(test.create(testSpace), test.requireOk)
+
+	require.Nil(test.T(), err)
+	require.NotNil(test.T(), newSpace)
+	updatedListOfSpaces, newCount, _ := test.list(nil, nil)
+
+	assert.True(test.T(), newCount > orgCount)
```

Comment:
yes , that's what it was. but the count was increasing by more than 1 - which could be because of some other parallel test creating spaces.

```
--- FAIL: TestList (0.01s)
	space_test.go:110: Old count of spaces : 2 , new count of spaces : 5
	assertions.go:241: 
                        
	Error Trace:	space_test.go:111
		
	Error:		Not equal: 0x3 (expected)
			        != 0x5 (actual)
```